.changeset/wild-kids-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@espressive/prop-types": patch
+---
+
+chore(FDS-442): [Rollup] - Improve rollup config to handle code-only packages
```

Comment:
```suggestion
chore(FDS-442): [@espressive/prop-types] - Added a main (cjs) and modules (esm) entry point to the package
```
If this change was only to the rollup config, we would not need a Changeset because we do not ship the Rollup config. However, this update to the config changes the prop-types package and the changeset should describe the impact of that change.